python/examples/parse_bmp.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python
+import struct
+
+import veles.veles_api as vapi
+from veles import objects
+
+
+class BmpParser(object):
+    # TODO change a lot of chunks to items when they are supported
+    # and add some comments to fileds
+    # parse more variations of bitmap
+    def __init__(self):
+        self.client = vapi.VelesApi('127.0.0.1', 3135)
+        self.current_start = 0
+
+    def parse_files(self):
+        files = self.client.list_files()
+        for file in files:
+            if file.name.endswith('.bmp'):
+                self.parse_bmp(file)
+
+    def create_chunk(self, parent, name,
+                     start=None, end=None, size=None, comment=None):
```

Comment:
Maybe it would be nice to later add this method to "standard" veles library? Seems quite convenient.